File: Mukul Bhalla/Day 3/Day 3 A.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        ans = []
        n = len(nums)
        for i in range(0, n):
            for j in range(i+1, n):
                if (nums[i]+nums[j] == target):
                    ans.append(i)
                    ans.append(j)
        return ans

File: Saatvik Srivastava/Day 3/Day3A.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int arr[] = new int[2];
        HashMap<Integer,Integer>  mp = new HashMap<>();
        for(int i=0;i<nums.length;i++){
            if(mp.containsKey(target-nums[i])){
                arr[0] = i;
                arr[1] = mp.get(target-nums[i]);
            }
            mp.put(nums[i],i);
        }
        return arr;
    }
}
